js/Level.js

```diff
@@ -44,5 +51,42 @@ export class Level {
         this.entities.forEach(element => {
             element.draw(context, this.offset)
         });
+        this.drawOverlay(game, context);
+    }
+    drawOverlay(game, context) {
+        //avatar life
+        context.fillStyle = "#BEBEBE";
+        context.fillRect(5, 5, 50, 50);
+        context.fillStyle = "#FFFFFF";
+        context.moveTo(5, 5);
+        context.lineTo(55, 5);
+        context.lineTo(55, 55);
+        context.lineTo(5, 55);
+        context.lineTo(5, 5);
+        context.stroke();
+        context.fillStyle = "#000000";
+        context.font = "12px Tahoma";
+        var scoreTxt = "Score: " + this.score;
+        var timeTxt = "Time: " + this.time;
+
+
+        // context.fillText(player.pos, game.width - 450, 20, 120);
+
+        context.strokeText(scoreTxt, game.width - 150, 20, 120);
+        context.strokeText(timeTxt, game.width - 150, 40, 120);
+
+        if (this.time <= 0 || this.health <= 0) {
+            if (time < -3) {
+                context.fillStyle = "#FFFFFF";
+                context.fillRect(0, 0, game.width, game.height);
+            } else {
+                context.fillStyle = "#F0F0FF";
+                context.font = "92px Tahoma";
+                context.fillText("GAME OVER", game.width / 2 - 300, game.height / 2, 600);
```

Comment:
```suggestion
                context.textAlign = "center";
                context.fillText("GAME OVER", game.width / 2 , game.height / 2);
```
so ist der text im echten center